app/controllers/api/v1/users/unlocks_controller.rb

```diff
@@ -0,0 +1,28 @@
+class Api::V1::Users::UnlocksController < Devise::UnlocksController
+  # GET /resource/unlock/new
+  # def new
+  #   super
+  # end
+
+  # POST /resource/unlock
+  # def create
+  #   super
+  # end
+
+  # GET /resource/unlock?unlock_token=abcdef
+  # def show
+  #   super
+  # end
+
+  # protected
+
+  # The path used after sending unlock password instructions
+  # def after_sending_unlock_instructions_path_for(resource)
+  #   super(resource)
+  # end
+
+  # The path used after unlocking the resource
+  # def after_unlock_path_for(resource)
+  #   super(resource)
+  # end
```

Comment:
Why do you guys have so much commented out code? It's always a good idea to leave out commented out code since git keeps track of changes you can always view changes in git history. (Unless you intend on using it soon).